dd-sdk-android/src/test/kotlin/com/datadog/android/tracing/internal/TracesFeatureTest.kt

```diff
@@ -246,6 +252,51 @@ internal class TracesFeatureTest {
         assertThat(endpointUrl).isSameAs(endpointUrl2)
     }
 
+    @Test
+    fun `it will register and unregister the provided plugin when initialise and stop called`(
```

Comment:
Just to be safe, I'd split this in 2 tests : verifying that initialize only calls register, and stop only calls unregsiter.